packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/translations/Messages_fr_FR.json

```
@@ -102,6 +102,7 @@
   "telephony_number_feature_ovh_pabx_step_rule_ivr": "Lancer un menu interactif",
   "telephony_number_feature_ovh_pabx_step_rule_ivr_explain": "Permet de proposer à l'appelant un menu avec lequel il pourra interagir grâce aux touches de son téléphone.",
   "telephony_number_feature_ovh_pabx_step_rule_ivr_menu_to_launch": "Menu à lancer",
+  "telephony_number_feature_ovh_pabx_step_rule_ivr_menu_already_added": "Menu déjà ajoutés",
```

Not needed to add it into `Messages_**_**.json`.

**question:** Shouldn't be this instead?
```suggestion
  "telephony_number_feature_ovh_pabx_step_rule_ivr_menu_already_added": "Menu déjà ajouté",
```